src/centerpoint/houston.py

```diff
@@ -79,18 +81,28 @@ def pennywise_conserve_saver6(x):
   return base + (0.03 + 0.0357) * 500 + 0.0357 * 500 + (0.119 + 0.0357) * (x - 1000)
 
 def test(f):
-  kwh = [103, 185, 337, 615, 861, 1200, 400, 350, 200, 200, 200, 150]
+  kwh = [int(x) for x in sys.argv[1:]]
   return sum(f(x) for x in kwh)
 
-print(test(discount_power_economy12))
-print(test(gexa_my_choice12))
-print(test(discount_power_easy24))
-print(test(power_express_summer24))
-print(test(infuse_savings6))
-print(test(pioneer_choice24))
-print(test(green_mountain_conserve12))
-print(test(discount_super_saver))
-print(test(pioneer_simple24))
-print(test(infinite_tremendous))
-print(test(reliant_secure))
-print(test(pennywise_conserve_saver6))
+function_dict = {
```

Comment:
Can this be generated using reflection instead? The goal is to make the process of adding a new plan simply adding the new function without needing to update any other configuration or mapping.

Reply:
Was interested in #3 but saw this PR so thought I'd just leave a comment: something like [this StackOverflow post](https://stackoverflow.com/questions/17380157/execute-all-functions-from-module) may help; It might be better architecture to start moving all the plans/functions into their own file, and have a "main" file that acts as the command-line interface.

Reply:
Yes, It might be better architecture to move all plans into their own file as @colbyguan said. I am going to move `test` and `all_estimates` functions to `main.py` file.